experiments/modelfree/noisy_actions_and_obs.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+function wait_proc {
```

Comment:
This is used in three files now (this, `attack_transfer.sh` and `baselines.sh`). I think time to refactor it into a separate file. For example, if we put `wait_proc` in a file `common.sh` then we could include that from other scripts by doing:
```bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
. ${DIR}/common.sh
```

Reply:
Done